rinja_derive_standalone/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "rinja_derive_standalone"
+version = "0.13.0"
+description = "Procedural macro package for Rinja"
+homepage = "https://github.com/rinja-rs/rinja"
+repository = "https://github.com/rinja-rs/rinja"
+license = "MIT/Apache-2.0"
+workspace = ".."
+readme = "README.md"
+edition = "2021"
+rust-version = "1.65"
+publish = false
+
+[features]
+default = ["__standalone"]
+__standalone = []
+config = ["serde", "basic-toml"]
```

Comment:
Missing the `:dep` part you mentioned. ;)

Reply:
Hehe, yeah. I leave this task to tomorrow-me. :D

Reply:
I approve this way of seeing things! :laughing: